lib/modules/manager/crow/extract.ts

```diff
@@ -0,0 +1,64 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { parseSingleYaml } from '../../../util/yaml';
+import { getDep } from '../dockerfile/extract';
+import type { ExtractConfig, PackageFileContent } from '../types';
+import type { CrowConfig } from './types';
+
+function crowVersionDecider(CrowConfig: CrowConfig): (keyof CrowConfig)[] {
+  const keys = ['clone', 'steps', 'pipeline', 'services'];
+  return Object.keys(CrowConfig).filter((key) =>
+    keys.includes(key),
+  ) as (keyof CrowConfig)[];
```

Comment:
```suggestion
function crowVersionDecider(CrowConfig: crowConfig): (keyof CrowConfig)[] {
  const keys = ['clone', 'steps', 'pipeline', 'services'];
  return Object.keys(crowConfig).filter((key) =>
    keys.includes(key),
  ) as (keyof CrowConfig)[];
```
please use caml case for variables, otherwise it's confusing to read and distinguish from types.

Reply:
The object you highlighted is a Type and Type imports from other managers also start with a capital letter? Are you sure that's the way to go here?

```
import type { CrowConfig } from './types';
```